pages/dynamic-portal.tsx

```diff
@@ -505,24 +476,22 @@ const DynamicTemplates: NextPageWithLayout<Props> = ({
             githubUrl="https://github.com/FlatFilers/hcm-show-config/blob/main/src/workflows/dynamic-templates/index.ts"
             features={features}
           />
+          {/* TODO: Add spinner while embed is loading */}
+          {showSpace && (
+            <div className="absolute top-0 right-0 w-full h-full bg-black/60">
+              <div className="relative mt-16 mx-auto max-w-7xl">
+                <XCircleIcon
+                  className="h-7 w-7 absolute top-[-32px] right-[-20px] hover:cursor-pointer text-white"
+                  onClick={() => setShowSpace(false)}
+                >
+                  X
+                </XCircleIcon>
+                <div ref={modalRef}>{component}</div>
+              </div>
+            </div>
+          )}
```

Comment:
Since they have an X in the modal let's get rid of ours

Reply:
Same for embed

Reply:
Added suggestion in slack